nixos/doc/manual/release-notes/rl-1709.xml

```diff
@@ -187,6 +187,17 @@ rmdir /var/lib/ipfs/.ipfs
       have therefore been removed.
     </para>
   </listitem>
+  <listitem>
```

Comment:
The salient point that `null` now implies mutable timezone management? If so, maybe say only that. The remainder of the entry reads to me as basically saying that something *hasn't* changed (UTC being the default timezone).